workers/unity/Packages/io.improbable.gdk.core/Dynamic/ComponentDatabase.cs

```diff
@@ -1,43 +1,46 @@
 using System;
 using System.Collections.Generic;
+using System.Linq;
+using Improbable.Gdk.Core.Commands;
 
 namespace Improbable.Gdk.Core
 {
-    internal static class ComponentDatabase
+    public static class ComponentDatabase
     {
-        public static Dictionary<uint, IDynamicInvokable> IdsToDynamicInvokers { get; }
+        internal static Dictionary<uint, IComponentMetaclass> Metaclasses { get; }
 
-        public static Dictionary<Type, uint> ComponentsToIds { get; }
+        private static Dictionary<Type, uint> ComponentsToIds { get; }
 
-        public static Dictionary<Type, uint> SnapshotsToIds { get; }
+        private static Dictionary<Type, uint> SnapshotsToIds { get; }
 
         static ComponentDatabase()
```

Comment:
do we not want an access modifier here

Reply:
Static constructors can't have access modifiers 😉